hare/flows_test.go

```diff
@@ -462,10 +314,11 @@ func Test_multipleCPs(t *testing.T) {
 		}
 		return true
 	}, 5*time.Second, 10*time.Millisecond)
+	layerDuration := 3 * time.Second
```

Comment:
This is quite a significant change from the previous value of 250ms, and I suspect its causing the test to not test what it intended to test which is multiple consensus processes running concurrently.

Reply:
Also why change this just for this test and not for `Test_multipleCPsAndIterations`

Reply:
> and I suspect its causing the test to not test what it intended to test which is multiple consensus processes running concurrently.

nope. it still runs multiple CPs concurrently. 3 second of layer duration is still less than 1 sec of wakeup delta + 1 preround + 4 hare rounds.

Reply:
> Also why change this just for this test and not for Test_multipleCPsAndIterations

that was a mistake. will fix

Reply:
the reason why i increased it is to lessen the artificial pressure cramping all the clock tick together.
i don't see why we should test round duration in the seconds but layer time in the 1/4 second.

Reply:
I'm not really understanding what you mean by
>artificial pressure cramping all the clock tick together

With the previous layer time this test would have been running consensus processes concurrently for most of the test,  with the new time that overlap of processes running concurrently has been reduced, so the second layer would overlap with the first but by the time the third layer starts the first will have finished. So although it may still be testing concurrent consensus processes I'm not sure it's testing them to the same degree as before.

Reply:
you are right. i wasn't considering all 3 processes running at the same time. will fix.